wa/workloads/jetstream/__init__.py

```diff
@@ -0,0 +1,90 @@
+#    Copyright 2014-2020 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+import re
+
+from wa import ApkUiautoWorkload, Parameter
+from wa.framework.exception import ValidationError, WorkloadError
+from wa.utils.types import list_of_strs
+from wa.utils.misc import unique
+
+
+class Jetstream(ApkUiautoWorkload):
+
+    name = 'jetstream'
+    package_names = ['com.android.chrome']
+    regex = re.compile(r'Jetstream Score ([\d.]+)')
+    tests = ['3d-cube-SP', '3d-raytrace-SP', 'acorn-wtb', 'ai-astar', 'Air', 'async-fs', 'Babylon', 'babylon-wtb',
+             'base64-SP', 'Basic', 'bomb-workers', 'Box2D', 'cdjs', 'chai-wtb', 'coffeescript-wtb', 'crypto',
+             'crypto-aes-SP', 'crypto-md5-SP', 'crypto-sha1-SP', 'date-format-tofte-SP', 'date-format-xparb-SP', 'delta-blue',
+             'earley-boyer', 'espree-wtb', 'first-inspector-code-load', 'FlightPlanner', 'float-mm.c', 'gaussian-blur', 
+             'gbemu', 'hash-map', 'jshint-wtb', 'json-parse-inspector', 
+             'json-stringify-inspector', 'lebab-wtb', 'mandreel', 'ML', 'multi-inspector-code-load', 'n-body-SP',
+             'navier-stokes', 'octane-code-load', 'octane-zlib', 'OfflineAssembler', 'pdfjs', 'prepack-wtb', 'raytrace',
+             'regex-dna-SP', 'regexp', 'richards', 'segmentation',
+             'splay', 'stanford-crypto-aes', 'stanford-crypto-pbkdf2', 'stanford-crypto-sha256', 'string-unpack-code-SP',
+             'tagcloud-SP', 'typescript', 'uglify-js-wtb', 'UniPoker']
+    description = '''
+    A workload to execute the jetstream web based benchmark
+
+    Test description:
+    1. Open chrome
+    2. Navigate to the jetstream website - https://browserbench.org/JetStream/
+    3. Execute the benchmark
+
+    known working chrome version 80.0.3987.149
+    '''
+    requires_network = True
+
+    def __init__(self, target, **kwargs):
+        super(Jetstream, self).__init__(target, **kwargs)
+        self.gui.timeout = 1500
```

Comment:
On my device this timeout was reached. (took ~33 mins to complete). Can this be raised to be greater then the timeout from the java side?